examples/scripts/spm_scipymin.py

```diff
@@ -0,0 +1,57 @@
+import pybop
+import pandas as pd
+
+# Form dataset
+Measurements = pd.read_csv("examples/scripts/Chen_example.csv", comment="#").to_numpy()
+dataset = [
+    pybop.Dataset("Time [s]", Measurements[:, 0]),
+    pybop.Dataset("Current function [A]", Measurements[:, 1]),
+    pybop.Dataset("Terminal voltage [V]", Measurements[:, 2]),
+]
+
+# Define model
+parameter_set = pybop.ParameterSet("pybamm", "Chen2020")
+model = pybop.models.lithium_ion.SPM(
+    parameter_set=parameter_set, options={"thermal": "lumped"}
+)
+
+# Fitting parameters
+parameters = [
+    pybop.Parameter(
+        "Negative electrode active material volume fraction",
+        prior=pybop.Gaussian(0.75, 0.05),
+        bounds=[0.6, 0.9],
+    ),
+    pybop.Parameter(
+        "Positive electrode active material volume fraction",
+        prior=pybop.Gaussian(0.65, 0.05),
+        bounds=[0.5, 0.8],
+    ),
+]
+
+# Define the cost to optimise
+signal = "Terminal voltage [V]"
+problem = pybop.FittingProblem(model, parameters, dataset, signal=signal, init_soc=0.98)
+cost = pybop.RootMeanSquaredError(problem)
+
+# Build the optimisation problem
+optim = pybop.Optimisation(cost=cost, optimiser=pybop.SciPyMinimize)
```

Comment:
I wonder if we should convert this to SciPyDifferentialEvolution once the corresponding PR is merged? As it's a global optimiser, we may want to recommend it more than SciPyMinimize. Thoughts?

Reply:
Sure, makes sense. I'll merge this one and then let's update in PR #132.